### PR TITLE
XP-4926 Publishing wizard - Unnamed dependencies are allowed to be pu…

### PR DESCRIPTION
**Pull request by vbradnitski**
…blished

### CODE REVIEW

File: modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/CheckContentsValidCommand.java [hunk 1]
```diff
@@ -29,11 +34,16 @@ public boolean execute()
             return true;
         }
 
+        final QueryExpr selectInvalidExpr = QueryExpr.from( LogicalExpr.
+            or( LogicalExpr.
+                    or( new NotExpr( CompareExpr.
+                        like( FieldExpr.from( ContentPropertyNames.DISPLAY_NAME ), ValueExpr.string( "*" ) ) ), CompareExpr.
```
Comment (runarmyklebust) [inline modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/CheckContentsValidCommand.java]: Why do you need the DISPLAYNAME = "*" here?

Comment (vbradnitski) [inline modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/CheckContentsValidCommand.java]: It's not DISPLAYNAME = "*", it's NotExpr from DISPLAYNAME = "*"

**Comment by runarmyklebust** [comment]
Ok, I think that the displayName != "" should be a part of the regular content.valid-check, waiting for TSI to mean something

**Comment by runarmyklebust** [comment]
Also mister, please add tests in ContentServiceImplTest_isValidContent for the different new scenarios

**Comment by runarmyklebust** [comment]
Please add the required checks in the "valid" check instead, so these new query-parts could be removed.

**Pull request closed without merging**